GunShipAlpha/src/Game.java

```
@@ -0,0 +1,157 @@
+import javax.swing.*;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import javax.imageio.ImageIO;
+import java.io.File;
+import java.io.IOException;
+import java.util.Random;
+import javax.swing.Timer;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.awt.geom.Point2D;
+
+public class Game extends Canvas {
+    private BufferedImage sprite;
+    static GameState gs = new GameState();
+    static MouseState ms = new MouseState();
+
+
+    public Game() {
+        loadSprite();
+        startGameLoop();
+        setupMouseListeners();
+    }
+
+    private void loadSprite() {
+        try {
+            sprite = ImageIO.read(new File("src/Images/GunShip.png"));
+
+        } catch (IOException e) {
+            e.printStackTrace();
```

make this "throw new RuntimeException(e)" rather than just print stack trace. You really want the program to crash when it hits a weird error rather than limp along in a busted state. You really only ever want to just print if you _know_ the program can continue in this state. without the sprite, for example, the game is just going to be fucked.

---

fixed